tests/test_type_constraints/test_unify.py

```diff
@@ -121,12 +124,12 @@ def test_same_forward_ref():
 def test_diff_forward_ref():
     fr1 = _ForwardRef('a')
     fr2 = _ForwardRef('b')
-    unify_helper(fr1, fr2, error_msg)
+    unify_helper(fr1, fr2, TypeFail("Attempted to unify forwardref  with non-ref"))
```

Comment:
Skip this test for now. The existing error message doesn't apply to this situation.